// Copyright 2021 The CPP Proto Builder Authors
//
// Licensed under the Apache License, Version 2.0 (the "License");
// you may not use this file except in compliance with the License.
// You may obtain a copy of the License at
//
//      http://www.apache.org/licenses/LICENSE-2.0
//
// Unless required by applicable law or agreed to in writing, software
// distributed under the License is distributed on an "AS IS" BASIS,
// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
// See the License for the specific language governing permissions and
// limitations under the License.

// READ: https://google.github.io/cpp-proto-builder

syntax = "proto2";

package proto_builder;

// A message intended for testing the case where a map contains a Message
// defined in a different file as its value type. This type should *only*
// be referenced in a map, not as a normal field, since that would defeat
// the purpose of the test!
message MapValueTestMessage {
  optional int32 one = 1;
}
